test/src/test/java/ai/lzy/test/PyApiTest.java

```diff
@@ -50,18 +50,10 @@ public void testExecFail() {
     }
 
     @Test
-    public void testEnvFail() {
```

Comment:
why?

Reply:
Because I deleted the possibility to pass custom yaml, as promised.